web-client/src/models/offers/index.ts

```diff
@@ -28,9 +30,11 @@ export interface IOffer extends firebase.firestore.DocumentData {
     firebase.firestore.DocumentData
   >;
   cavUserSnapshot: IUser;
+  requestSnapshot: IRequest | null;
   message: string;
   status: OfferStatus;
   createdAt?: firebase.firestore.Timestamp;
+  seenAt?: firebase.firestore.Timestamp;
```

Comment:
same as I mentioned in model at functions

Reply:
Fixed